src/test/java/ch/ennio/sileno/tutorial/fullstack/FullstackApplicationTests.java

```diff
@@ -8,6 +9,7 @@ class FullstackApplicationTests {
 
 	@Test
 	void contextLoads() {
+		Assertions.fail("Oops test failed");
```

Comment:
this assertion will fail and crash your branch.

Reply:
thanks for info